cumulusci/tests/util.py

```diff
@@ -44,12 +44,12 @@ def __init__(
 
 
 class DummyOrgConfig(OrgConfig):
-    def __init__(self, config=None, name=None):
+    def __init__(self, config=None, name=None, keychain=None):
```

Comment:
For completeness, let's make sure this has `global_org` too.

Reply:
7e25e3cd